### PR TITLE
Updated sidebar

### PR DESCRIPTION
**Pull request by jono-alderson**
- Added references to new pages
- Moved Product Sheet

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -33,6 +33,25 @@ module.exports = {
...
+                        },
+
+                        {
+							type: 'category',
+							label: 'Descriptions',
+							items: [
+								'features/seo-tags/descriptions/overview',
+								'features/seo-tags/descriptions/functional-specification',
+								'features/seo-tags/descriptions/api',
+							],
```
Comment (jcomack) [inline sidebars.js:53]: Indentation seems off.

File: sidebars.js [hunk 2]
```diff
@@ -162,6 +192,7 @@ module.exports = {
 				'standards/version-control-conventions',
 				'standards/productivity-tips-and-tricks',
 				'standards/cheatsheets',
+                'standards/yoast-seo-product-sheet',
```
Comment (jcomack) [inline sidebars.js:195]: Indentation seems off.

**Comment by jcomack** [comment]
Requires: https://github.com/Yoast/developer-docs/pull/57

File: sidebars.js (hunk 2 above)
Comment (jono-alderson) [reply sidebars.js:195]: Can you wrangle this from your side? Think it's a Windows thing; struggling to align!

File: sidebars.js (hunk 1 above)
Comment (jono-alderson) [reply sidebars.js:53]: Can you wrangle this from your side? Think it's a Windows thing; struggling to align!

Comment (jcomack) [reply sidebars.js:53]: Tabs vs spaces :P

**Pull request merged**